recipes/wasmedge/all/conanfile.py

```diff
@@ -41,29 +43,37 @@ def package_id(self):
         self.info.settings.compiler = self._compiler_alias
 
     def source(self):
-        tools.get(**self.conan_data["sources"][self.version][str(self.settings.os)][str(self.settings.arch)][self._compiler_alias][0],
+        get(self, **self.conan_data["sources"][self.version][str(self.settings.os)][str(self.settings.arch)][self._compiler_alias][0],
                   destination=self._source_subfolder, strip_root=True)
-        tools.download(filename="LICENSE",
-                       **self.conan_data["sources"][self.version][str(self.settings.os)][str(self.settings.arch)][self._compiler_alias][1])
+        download(self, filename="LICENSE",
+                       **self.conan_data["sources"][self.version][str(self.settings.os)][str(self.settings.arch)][self._compiler_alias][1])  # noqa: E128
 
     def package(self):
         self.copy("*.h", src=os.path.join(self._source_subfolder, "include"), dst="include", keep_path=True)
         self.copy("*.inc", src=os.path.join(self._source_subfolder, "include"), dst="include", keep_path=True)
 
         srclibdir = os.path.join(self._source_subfolder, "lib64" if self.settings.os == "Linux" else "lib")
         srcbindir = os.path.join(self._source_subfolder, "bin")
-
-        self.copy("wasmedge_c.lib", src=srclibdir, dst="lib", keep_path=False)
-        self.copy("wasmedge_c.dll", src=srcbindir, dst="bin", keep_path=False)
-        self.copy("libwasmedge_c.so*", src=srclibdir, dst="lib", keep_path=False)
-        self.copy("libwasmedge_c.dylib", src=srclibdir,  dst="lib", keep_path=False)
+        if Version(self.version) >= "0.11.1":
+            self.copy("wasmedge.lib", src=srclibdir, dst="lib", keep_path=False)
```

Comment:
Would be nice to sneak in `conan.tools.files.copy` here to get some more progress updating it 🙏